tests/runtime/probe

```diff
@@ -210,7 +210,7 @@ WILL_FAIL
 
 NAME kprobe_warn_missing_probes
 PROG kprobe:vfs_read,kprobe:nonsense { exit(); }
-EXPECT WARNING: could not attach probe kprobe:nonsense, skipping.
+EXPECT WARNING: could not attach probe kprobe:nonsense (kprobe:vfs_read,kprobe:nonsense), skipping.
```

Comment:
why have the warning show both the probes if only one is skipped?

Reply:
Ah, I probably needed it for debugging the test. I can clean it up...